transport/src/main/java/io/netty/channel/ChannelHandler.java

```diff
@@ -377,10 +370,9 @@ default void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise
      * that are pending.
      *
      * @param ctx               the {@link ChannelHandlerContext} for which the flush operation is made
-     * @throws Exception        thrown if an error occurs
      */
     @Skip
-    default void flush(ChannelHandlerContext ctx) throws Exception {
+    default void flush(ChannelHandlerContext ctx) {
         ctx.flush();
```

Comment:
What happens now if `flush` fails?

Reply:
We log and close

Reply:
` ctx.flush()` can't fail (not sure if this was what you asked for)

Reply:
@chrisvest any more questions ?